pkg/controller/direct/common/refs.go

```diff
@@ -165,42 +152,3 @@ func (r *refNormalizer) VisitField(path string, v any) error {
 
 	return nil
 }
-
-// RefineMonitoringAlertPolicyRefForAlertChart refine the alertPolicy format because MonitoringDashboard's AlertChart has a specific format requirement:
-// "The format is: projects/[PROJECT_ID_OR_NUMBER]/alertPolicies/[ALERT_POLICY_ID]"
-func (r *refNormalizer) RefineMonitoringAlertPolicyRefForAlertChart(external string) (string, error) {
```

Comment:
`RefineMonitoringAlertPolicyRefForAlertChart` and `RefineMonitoringAlertPolicyRefForIncidentList` are moved to Monitoring reference file.